example-scripts/bash-scripts/full-harvest-examples/1.13-1.15-examples/example-openalex/openalex-to-vivo.datamap.xsl

```diff
@@ -0,0 +1,1513 @@
+<!--
+  Copyright (c) 2010-2011 VIVO Harvester Team. For full list of contributors, please see the AUTHORS file provided.
+  All rights reserved.
+  This program and the accompanying materials are made available under the terms of the new BSD license which accompanies this distribution, and is available at http://www.opensource.org/licenses/bsd-license.html
+-->
+<!-- Header information for the Style Sheet
+	The style sheet requires xmlns for each prefix you use in constructing
+	the new elements
+-->
+
+<xsl:stylesheet version = "2.0"
+                xmlns:xsl = 'http://www.w3.org/1999/XSL/Transform'
+                xmlns:rdf = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+                xmlns:rdfs = 'http://www.w3.org/2000/01/rdf-schema#'
+                xmlns:core = 'http://vivoweb.org/ontology/core#'
+                xmlns:score = 'http://vivoweb.org/ontology/score#'
+                xmlns:foaf = 'http://xmlns.com/foaf/0.1/'
+                xmlns:bibo = 'http://purl.org/ontology/bibo/'
+                xmlns:obo = 'http://purl.obolibrary.org/obo/'
+                xmlns:vitro = 'http://vitro.mannlib.cornell.edu/ns/vitro/0.7#'
+                xmlns:vcard = 'http://www.w3.org/2006/vcard/ns#'
+                xmlns:kdsf-vivo = 'http://lod.tib.eu/onto/kdsf/'
+                xmlns:node-publication='http://vivo.example.com/harvest/aims_users/fields/publication/'
+                xmlns:fn='http://www.w3.org/2005/xpath-functions'
+                xmlns:functx='http://www.functx.com'
+                xmlns:vivo-oa='http://lod.tib.eu/onto/vivo-oa/'
```

Comment:
TIB specific namespace

Reply:
I will remove all the TIB specific things..